robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino_base_plan.urdf.xacro

```diff
@@ -19,6 +20,10 @@
     </plugin>
   </gazebo>
 
+  <xacro:shell name="base_shell" parent="base_link">
```

Comment:
yeah, i did it on the robot, but apparently forgot to push it (and the robot's already gone); pls remind me in Freiburg again :D na, I shouldn't forget to push it.